_posts/2021-08-29-mosaic-fusion.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Mosaic Image Fusion
+author: Volker Hilsenstein and Genevieve Buckley
+tags: [life science]
+theme: twitter
+---
+{% include JB/setup %}
+
+## Executive Summary
+
+This blogpost shows a case study, where a researcher uses Dask for mosaic image fusion - stitching multiple smaller images together seamlessly into a very large field of view. Full code examples are available on GitHub from the `DaskFusion` repository: https://github.com/VolkerH/DaskFusion
+
+This work was done by Volker Hilsenstein, a scientific software developer at EMBL in Theodore Alexandrov's lab. Special thanks go to Marvin Albert for valuable discussions and advice. Genevieve Buckley wrote the blogpost.
+
+## The problem
+
+Volker works on [spatial metabolomics](https://www.ebi.ac.uk/training/online/courses/metabolomics-introduction/what-is/) and bio-image analysis.
```

Comment:
Maybe @VolkerH could add a sentence here describing what sort of samples his lab looks at, etc.